src/tabs/AOCTab/days/day3/Day3Solution.js

```diff
@@ -1,14 +1,136 @@
 const Day3Solution = (props) => {
+    // Parses file, separating string into individual values
   const input = props.input
-  console.log("Day 3 input: " + input); 
+  var binary_input = input.split("\r\n");

```

Comment:
A few comments. One, when we have a variable like this in JS we always want to use `let` instead of `var`, two I don't really care for this PR, but the variable naming conventions are `camelCaseVariables` not `snake_case_variables` but at the same time I know this is based on Python code so it's okay. Finally, if you ran this on a Mac computer, it actually wouldn't work! This is because the files are encoded as just `\n` instead of `\r\n` like they are on your windows-based computer, so you would actually do something like `.split(/\r?\n/)` using a regular expression to take care of both. I did a simple google search for that don't worry, you definitely don't have to memorize that, but just remember that there is this difference between windows and mac when it comes to newlines!